test/test_runner.go

```diff
@@ -786,6 +795,11 @@ func (r *TestRunner) parseAndCheckImport(
 
 	env.CheckerConfig.ContractValueHandler = contractValueHandler
 
+	err = setupEVMEnvironment(r.backend.blockchain.NewScriptEnvironment(), env)
```

Comment:
Not sure, but should this create a new environment on each call? Can the environment constructed in `TestRunner.initializeEnvironment` above be reused?

Reply:
That's a good point. Creating a new environment on each call will certainly add unnecessary overhead. I will try to work-around that.

Reply:
@turbolent Updated in https://github.com/onflow/cadence-tools/pull/338/commits/e0f8a9b1ce76ee8b403e03403c8eaab36494916e :pray:

Reply:
Nice!